src/apps/users/services/core.py

```diff
@@ -29,6 +29,14 @@ def __init__(self, session) -> None:
         self._cache: PasswordRecoveryCache = PasswordRecoveryCache()
         self.session = session
 
+    def get_localized_subject(self, language: str) -> str:
+        subjects = {
```

Comment:
please, rename function to `get_locallized_email_subject`, just to be clear that it is not related to platform subject concept